FixedArray limit reached (512)
BUILD: 2023-01-04-21-43-18--hlsdl-64
EXCEPTION:

Called from dn.Tweenie.create_ (dn/Tweenie.hx line 423)
Called from sample.WaterPond.fixedUpdate (sample/WaterPond.hx line 81)
Called from Game.fixedUpdate (Game.hx line 334)
Called from dn.$Process._doFixedUpdate (dn/Process.hx line 550)
Called from dn.$Process._doFixedUpdate (dn/Process.hx line 559)
Called from dn.$Process.updateAll (dn/Process.hx line 669)
Called from Boot.update (Boot.hx line 62)
Called from Boot.update (Boot.hx line 58)
CALL:

Called from $App.onCrash (App.hx line 63)
Called from $App.onCrash (App.hx line 57)
Called from Boot.update (Boot.hx line 69)
Called from hxd.App.mainLoop (hxd/App.hx line 193)
Called from hxd.$System.mainLoop (hxd/System.hl.hx line 76)
Called from hxd.$System.runMainLoop (hxd/System.hl.hx line 158)
Called from hxd.$System.runMainLoop (hxd/System.hl.hx line 142)
Called from haxe.$Timer.~delay.0 (C:\HaxeToolkit\haxe\std/haxe/Timer.hx line 144)
Called from haxe.$Timer.~__constructor__.0 (C:\HaxeToolkit\haxe\std/haxe/Timer.hx line 76)
Called from sys.thread.EventLoop.loop (C:\HaxeToolkit\haxe\std/sys/thread/EventLoop.hx line 177)
Called from sys.thread._Thread.$Thread_Impl_.processEvents (C:\HaxeToolkit\haxe\std/hl/_std/sys/thread/Thread.hx line 62)